symplyphysics/definitions/amount_energy_from_mass_and_temperature.py

```diff
@@ -10,21 +10,23 @@
 # m - body mass
 # t2 - final temperature
 # t1 - initial temperature
-amount_energy, specific_heat_capacity, body_mass, final_temperature, initial_temperature = symbols(
-    'amount_energy specific_heat_capacity body_mass final_temperature initial_temperature'
+# Note: specific heat capacity C is parameter indicates how much energy in joules
+# must be expended to heat a unit mass of a substance by one unit temperature.
```

Comment:
Also add a note about resulting amount of energy. Is it the generated energy or consumed energy?